app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -43,7 +43,7 @@ class Api::V2::RepositoriesController < Api::V2::ApiController
     param :erratum_id, String, :desc => N_("Id of an erratum to find repositories that contain the erratum")
     param :rpm_id, String, :desc => N_("Id of a package to find repositories that contain the rpm")
     param :library, :bool, :desc => N_("show repositories in Library and the default content view")
-    param :content_type, String, :desc => N_("limit to only repositories of this time")
+    param :content_type, String, :desc => N_("limit to only repositories of type 'yum', 'puppet', or 'docker'")
```

Comment:
@beav, @daviddavis, well..., i guess the changes to this file are kinda outside the scope of the rest of the PR?